library/src/main/java/com/xwray/groupie/DiffTask.java

```diff
@@ -13,20 +13,25 @@
  * in a background thread. This task will call {@link AsyncDiffUtil.Callback#onDispatchResult(Collection)}
  * passing the new list just before dispatching the diff result to the provided
  * {@link DiffUtil.Callback} so that the new list.
- * <p>This task is executed via {@link AsyncDiffUtil#calculateDiff(Collection, DiffUtil.Callback)}.
+ * <p>This task is executed via {@link AsyncDiffUtil#calculateDiff(Collection, DiffUtil.Callback, OnAsyncUpdateListener)}.
  */
 class DiffTask extends AsyncTask<Void, Void, DiffUtil.DiffResult> {
     private final DiffUtil.Callback diffCallback;
```

Comment:
Since you already added `@Nullable` to `onAsyncUpdateListener`, could you please add `@NonNull` to `diffCallback` and `asyncListDiffer`?

Reply:
done